canonical-websites/www.ubuntu.com
# Bootstack webinar and nav

## Done

 - Moved managed cloud up one level in the hierarchy
 - Added/updated redirects
 - Updated webinar
 - Added some Google events to external links

## QA

Check `/cloud/managed-cloud` (and check that `/cloud/managed-cloud` redirects)

[Brief](https://docs.google.com/document/d/1kLApndeYU2h4TRs29gvKRqfe6vZtIi4qvjB8YUS6e24/edit?ts=589c5407)
[Copy](https://docs.google.com/document/d/1g38PPPhEP0Z66eM_gezfUyUwAr5e9vGfTHJDWV8gZlE/edit#)

NOTE:
I had to sanitise some files, turn of whitespace changes for a clearer diff
